Arduino/MiP_ProMini_Pack_Library/examples/DisconnectApp/DisconnectApp.ino

```diff
@@ -0,0 +1,42 @@
+#include <mip.h>
+#include <queue.h>
```

Comment:
Can this and the previous #include before the copyright message just be deleted?

Reply:
I didn't even notice those.  Will delete.